test/lib/babel-plugin-import-rewrite.js

```diff
@@ -86,3 +86,5 @@ describe("rewrite", () => {
 		expect(rewrite("../../src/lol", aliases)).toBe("../../dist/lol")
 	})
 })
+
+export default {}
```

Comment:
How come the test files have this empty export default?

Reply:
they have this because the vscode plugin for running Mocha tests gets real confused if it is using `-r esm` and the test has no default export.